src/Glpi/Form/AccessControl/FormAccessControlManager.php

```diff
@@ -92,6 +92,18 @@ public function getActiveAccessControlsForForm(
         return array_values($controls);
     }
 
+    public function isAnonymousForm(Form $form): bool
```

Comment:
It would be more explicit.
```suggestion
    public function allowUnauthenticatedAccess(Form $form): bool
```